libqtile/widget/prompt.py

```diff
@@ -25,7 +25,7 @@
 #
 # THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 # IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVhENT SHALL THE
```

Comment:
No worries, we're glad to have people contribute, more developers means more gets done and it's much easier to check for things like this than to do the actual work on the issue.